portfolio.css

```diff
@@ -135,3 +139,349 @@ h2 {
+  line-height: 20px;
+  text-align: left;
+}
+
+.html {
+  border: 1px solid #091e42;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+}
+
+.bootstrap {
+  border: 1px solid #091e42;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+}
+
+.ruby {
+  border: 1px solid #091e42;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+}
+
+.seeButton {
+  padding: 15px;
+  background-color: #ff6b00;
+  border: 1px solid #ff6b00;
+  text-align: center;
+  text-decoration: none;
+  position: relative;
+  left: 14px;
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+}
+
+.projects {
+  display: grid;
+  height: 2436px;
+  grid-template-columns: 327px;
+  grid-template-rows: 386px 386px 386px 386px 386px 386px;
+  align-content: space-between;
+  padding-top: 24px;
+  padding-left: 15px;
+}
+
+.cards {
+  background-image: url(images/cardBack.png);
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  height: 386px;
+  width: 327px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  color: #fff;
+  text-align: left;
+  position: relative;
+  height: 90px;
+  width: 295px;
+  left: 13px;
+  top: 52px;
+}
+
+.cardPara {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  color: #fff;
+  text-align: left;
+  position: relative;
+  left: 13px;
+  margin-right: 10px;
+  margin-top: 25px;
+}
+
+.cardList {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  color: #fff;
+  list-style: none;
+  height: 44px;
+  width: 275px;
+  position: relative;
+  right: 35px;
+  margin-top: 0;
+}
+
+.cardHtml {
+  border: 1px solid #091e42;
+  padding: 13px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+  background-color: #ffffff3d;
+}
+
+.cardBootstrap {
+  border: 1px solid #091e42;
+  padding: 13px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+  background-color: #ffffff3d;
+}
+
+.cardRuby {
+  border: 1px solid #091e42;
+  padding: 13px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: left;
+  background-color: #ffffff3d;
+}
+
+.cardBtn {
+  padding: 15px;
```

Comment:
Please follow the button styling: Active, Hover, and Pressed (see design). It's always a good practice to follow a given design.